### DISCUSSION TITLE
TRPCClientError: The string did not match the expected pattern.

### DISCUSSION THREAD

**Discussion by patrickphat** [trpc/trpc#4178] [Q&A]
How to fix the error? I do not happen to find a way to debug it because it only show vaguely in the browser?
<img width="543" alt="image" src="https://user-images.githubusercontent.com/45748186/231686982-34038fcb-163d-40d6-9fb4-c2377c915cde.png">

**Comment by KATT** [comment]
Post reproduction repo or we can't help you, this doesn't look like it originates from a trpc server at all
